arduino_unit_tests/cc1120_regs.h

```diff
@@ -0,0 +1,75 @@
+#ifndef REGS_CC1120_H
+#define REGS_CC1120_H
+
+#define IOCFG3              0x00
```

Comment:
I would add a comment here saying these are the config registers.

Reply:
Will do